Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Rideau",
+    platforms: [.iOS(.v10)],
+    products: [
+        .library(
+            name: "Rideau",
+            targets: ["Rideau"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/immortal79/TransitionPatch.git", from: "1.0.3")
```

Comment:
I created 1.0.3 as well.
https://github.com/muukii/TransitionPatch/releases/tag/1.0.3

Could you change the url to my repo's url?

```suggestion
        .package(url: "https://github.com/muukii/TransitionPatch.git", from: "1.0.3")
```

Reply:
Done 👍